### DISCUSSION TITLE
Modify log.html possible?

### DISCUSSION THREAD

**Discussion by PeterMcLennan** [lllyasviel/Fooocus#2491] [Ideas]
I'd like the log to display a larger image alongside the metadata.  Is this possible?  Fully half of my display is unused on the RHS of the log file display.

**Comment by mashb1t** [comment]
You can modify the code of Fooocus anytime, feel free to fork it and adjust the html in private_logger.py or add css to it, then create a PR back to this repo for a review and merge for all people using it.

**Reply by PeterMcLennan** [reply]
Thanks.  Good to know it's possible.  
Beyond my skill set, however. : (

Thanks for your prompt response.
